contribute/create-pull-request.md

```diff
@@ -48,6 +48,13 @@ Pull requests that add or modify unit tests that are written in Jest must adhere
 - Don't add snapshots tests. We are incrementally removing existing snapshot tests, we don't want more.
 - If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), unless you’re fixing a bug. Bug fixes usually shouldn't include any bigger refactoring, so it’s ok to skip migrating the test to RTL.
 
+Pull requests that creates new UI components must adhere to these accessibility guidelines:
+
+- Use semantic HTML.
+- Avoid using aria roles, labels and other accessibility attributes incorrectly. Accessibility attributes should only be used when semantic HTML doesn't satisfy your use case.
+- Use the Grafana theme palette for styling; contains colors with good contrast which aids accessibility.
```

Comment:
```suggestion
- Use the Grafana theme palette for styling. It contains colors with good contrast which aids accessibility.
```

We should also point the user to where can they learn how to use the theme palette.

Reply:
I think @torkelo is working on a doc for the new theme. Will do that once it's published.